src/components/transport_manager/src/usb/libusb/usb_handler.cc

```diff
@@ -485,13 +508,13 @@ void UsbHandler::SubmitControlTransfer(
   const libusb_request_type request_type = LIBUSB_REQUEST_TYPE_VENDOR;
 
   libusb_endpoint_direction endpoint_direction;
-#ifdef OS_WINCE
+#ifdef OS_WIN32
```

Comment:
There have many macro selection conditions for OS_WIN32, please adjust the code for more easier reading.